src/interfaces/blog.interfaces.tsx

```diff
@@ -0,0 +1,37 @@
+import { IApiResponse } from './api-response.interface';
+
+export interface IBlogListApiResponse extends IApiResponse {
+	value: IBlogList[];
+}
+
+export interface IBlogList {
```

Comment:
The fields do not match 

https://github.com/Golf-Course-Project/course-service/blob/main/CourseService/Entities/Blog.cs